src/tests/unit/platform/test_alertsv7_api.py

```diff
@@ -260,22 +262,18 @@ def test_query_alert_facets(cbcsdk_mock):
     def on_post(url, body, **kwargs):
         assert body["query"] == "Blort"
         t = body["criteria"]
-        assert t["workflow"] == ["OPEN"]
+        assert t["minimum_severity"] == ['3']
         t = body["terms"]
         assert t["rows"] == 0
-        assert t["fields"] == ["REPUTATION", "STATUS"]
-        return {"results": [{"field": {},
-                             "values": [{"id": "reputation", "name": "reputationX", "total": 4}]},
-                            {"field": {},
-                             "values": [{"id": "status", "name": "statusX", "total": 9}]}]}
-
+        assert t["fields"] == ["type"]
+        return GET_ALERT_FACET_RESP
     cbcsdk_mock.mock_request('POST', "/api/alerts/v7/orgs/test/alerts/_facet", on_post)
     api = cbcsdk_mock.api
 
-    query = api.select(Alert).where("Blort").set_workflows(["OPEN"])
-    f = query.facets(["REPUTATION", "STATUS"])
-    assert f == [{"field": {}, "values": [{"id": "reputation", "name": "reputationX", "total": 4}]},
-                 {"field": {}, "values": [{"id": "status", "name": "statusX", "total": 9}]}]
+    query = api.select(Alert).where("Blort").add_criteria("minimum_severity", "3")
+    f = query.facets(["type"])
+    assert f == [{'field': 'type', 'values': [{'id': 'WATCHLIST', 'name': 'WATCHLIST', 'total': 1916},
+                                              {'id': 'CB_ANALYTICS', 'name': 'CB_ANALYTICS', 'total': 41}]}]
```

Comment:
Line 309,  "alert_notes_present": ["True"] is invalid.  It's a singleton boolean so must be 
set_alert_notes_present(True)
to generate
{"criteria": {"alert_notes_present": true}